constants/utils/index.ts

```diff
@@ -4,20 +4,21 @@
  * SPDX-License-Identifier: AGPL-3.0-only
  */
 
-export const FOLDER_VIEW = {
-	search_folder: 'search folder',
-	tag: 'tag',
-	conversation: 'conversation',
-	message: 'message',
-	contact: 'contact',
-	document: 'document',
-	appointment: 'appointment',
-	virtual_conversation: 'virtual conversation',
-	remote_folder: 'remote folder',
-	wiki: 'wiki',
-	task: 'task',
-	chat: 'chat'
-};
+// eslint-disable-next-line no-shadow
+export enum FOLDER_VIEW {
```

Comment:
https://dev.to/ivanzm123/dont-use-enums-in-typescript-they-are-very-dangerous-57bh

This is the first article I found who spoke about enums.

Reply:
done, thank you